### PR TITLE
Document the current status of the e-puck 2

### PR DESCRIPTION
**Pull request by fabienrohrer**
Dual PR: https://github.com/omichel/webots/pull/6930

### CODE REVIEW

**Comment by fabienrohrer** [comment]
@tn12787 Could you review english here, please?

**Comment by fabienrohrer** [comment]
Thank you, Tom

**Comment by fabienrohrer** [comment]
👍

File: guide/epuck.md [hunk 1]
```diff
@@ -135,6 +135,28 @@ The resolution of the camera was limited to 52x39 pixels, as this is the maximum
...
 
+### e-puck 2
+
+%figure "e-puck 2 model"
+
+![epuck2.png](images/robots/epuck/epuck2.png)
+
+%end
+
+The [e-puck 2](http://www.gctronic.com/e-puck2.php) has been released in 2018.
```
Comment (omichel) [inline guide/epuck.md]: The correct spelling is "e-puck2", not "e-puck 2".

File: guide/epuck.md [hunk 2]
```diff
@@ -135,6 +135,28 @@ The resolution of the camera was limited to 52x39 pixels, as this is the maximum
...
+The e-puck 2 runs linux and is equipped with a WiFi chipset, among other components.
```
Comment (omichel) [inline guide/epuck.md]: It doesn't run Linux. This is wrong.

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**